src/components/Carousel.js

```diff
@@ -653,14 +653,17 @@ export default class Carousel extends Component {
   }
 
   render() {
+    const { value, children, slides } = this.props;
+    const lastSlideIndex = slides ? slides.length - 1 : React.Children.count(children) - 1;
+
     return (
       <div
         className={classnames('BrainhubCarousel', this.getProp('className'))}
         ref={el => this.node = el}
       >
-        {this.renderArrowLeft()}
+        {value > 0 && this.renderArrowLeft()}
```

Comment:
In many cases we would rather have a disabled button instead of no button at all. A class for a disabled arrow would be much better solution as it gives a developer a way to customize this behaviour.

Reply:
Can we set the disabled property of button based on the value (value > 0) the button statys disabled and vice versa for right arrow if `value === lastSlideIndex`.

Will this work or should we also be adding any css class in the component.

Reply:
@Lukasz-pluszczewski is sick now so you have to wait (about one week) until he recovers or I or somebody else manages to figure out the answer

Reply:
You can add an additional CSS class if you want, but disabled button can already be styled as needed so I think that's enough.